apps/dashboard/src/components/contract-components/contract-publish-form/default-factory.tsx

```diff
@@ -78,24 +81,24 @@ export const DefaultFactory: React.FC<DefaultFactoryProps> = ({
           />
         </FormControl>
 
-        {shouldShowExtensionsParamInput && (
+        {shouldShowModulesParamInput && (
           <>
             <Flex flexDir="column" gap={2} mt={4}>
-              <Heading size="title.md">Extension Param</Heading>
+              <Heading size="title.md">Modules Param</Heading>
```

Comment:
May be just "Module Param" instead of "Modules Param" here? @kumaryash90

Reply:
since it's an array input, called `_modules` on the contract, wrote it as Modules Param, or the param that takes modules